blaze/tests/test_interactive.py

```diff
@@ -201,6 +201,19 @@ def test_table_resource():
         assert isinstance(t.data, CSV)
         assert into(list, compute(t)) == into(list, csv)
 
+def test_explicit_dshape():
+    ds = dshape("var * {a: ?float64, b: ?string, c: ?float32}")
+    s = """a,b,c
```

Comment:
could we use `textwrap.dedent` on this? having strings go back to 0 indentation makes it harder to follow the flow of code when scrolling though a file.